### ISSUE TITLE
[FEATURE REQUEST] An option to not automatically reload a Tab upon switching

### ISSUE THREAD

**Issue by ChiroIp** [rNeomy/auto-tab-discard#169]
Have the option to show a Reload button, the description of the discarded page, the memory used at the moment of the discharge, a thumbnail instead of the automatic reload.

**Comment by Josehkn** [comment]
Second to this, i wanna see a thumbnail of the whole page like The Great Suspension does in Chrome, would love that feature.

**Comment by sagz** [comment]
Yes please!

**Comment by chrislockard** [comment]
+1! This is a very useful feature when I'm cycling through discarded tabs deciding which to restore or which to close in Brave. I miss this functionality on Firefox.

**Comment by bkmdev** [comment]
I completely agree with this thread 👍   I currently use The Great Suspender (TGS) on Chrome and finally found this FF extension which seems to come closest in functioning as a counterpart in FF, but do find that not being to choose when to restore is problematic (as a workaround I've had to use an empty-ish "parking tab" to not "set off" 💣 💥 any suspended tabs, but this is not ideal 🙁 ).  

I'd highly recommend following TGS's implementation, ie: do not resume any tabs until one clicks anywhere on the suspended tab, also show (at a minimum) the `favicon.ico|png` for the site with HTML title on the page (_prominently_ in \#h1 or \#h2 font size).  A state change icon on the page during load + memory saved stats shown on the discarded parked page with a thumbnail screenshot may be nice too (although I've found the thumbnail screenshot feature in TGS slows things down quite a bit- perhaps it can be done efficiently in FF though?)

**Comment by rafmsou** [comment]
I see this as a must, I use the keyboard to go through tabs and it's really annoying having all crossed tabs reloaded...

**Comment by moghaazi** [comment]
More than 8 months and they didn't add this important feature, I'm done with this extension I delete it

**Comment by chrislockard** [comment]
Given the recent security issues surrounding TGS (https://github.com/greatsuspender/thegreatsuspender/issues/1263) -  arguably this extension's primary competitor - adding this feature would bring ATD closer to feature parity and help capture wayward TGS users.

**Comment by frasty** [comment]
Yes, please add this feature. As a former TGS user this is what I miss the most.

**Comment by lucknaumann** [comment]
I never used TGS, but have used similar extensions. I used one that works much like this one does, but didn't reload the page. It was simply the page unloaded with a dimed overlay. The way to reload it was with the standard reload button. That's it. It wasn't fancy with an icon on the page or anything, but it was obvious the page wasn't loaded due to it being dimmed out. This seems a simplier yet effective implementation of this feature. However, as @rNeomy mentioned in issue #183 [comment-773882226](https://github.com/rNeomy/auto-tab-discard/pull/183#issuecomment-773882226) and [comment-774479660](https://github.com/rNeomy/auto-tab-discard/pull/183#issuecomment-774479660), there may be other reasons this could be difficult to implement.

Maybe just going to the last active tab, unless it's already discarded?

**Comment by gianpaj** [comment]
> Maybe just going to the last active tab, unless it's already discarded? Edit: Or the nearest active tab?

+1

**Comment by tcmeggs** [comment]
Another +1 for this, I'm a Firefox user and open many tabs. Frequently I will have a number of tabs for the same website that I need to go through that have all been snoozed, so I'll ctrl-pageup/pagedown through them to review. Herein lies the problem - I'm not sure if it's how Auto Tab Discard initiates the wake sequence, but if I flip through a number of snoozed tabs, **my computer is brought to its knees** and Firefox can take **several minutes to start responding again**. This is a huge inconvenience.

So not sure if this belongs on this feature request or as a bug, but I'm finding Auto Tab Discard untenable. Please let me know your thoughts or if any further details would be helpful.

Firefox 88.0.1 (64-bit) on Ubuntu 20.04.2.

**Comment by ivan-volnov** [comment]
@rNeomy +1. I want this feature too on Firefox.

**Comment by Amooleyna** [comment]
@rNeomy +1. I would like this feature too! It would make this extension even better!

**Comment by maddada** [comment]
I want to use this extension, but can't because it's missing this feature.
Also wish it would show a dimmed screenshot of the suspended page like The Great Suspender.
